src/test/java/connectify/model/company/CompanyEmailTest.java

```diff
@@ -0,0 +1,88 @@
+package connectify.model.company;
+
+import static connectify.testutil.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class CompanyEmailTest {
```

Comment:
I think we don't really have to split it up into Company Email and Person Email unless they have different types. But it's okay

Reply:
hi ryan, in my opinion the rationale for splitting up into CompanyEmail and Person Email is because we might incorporate different verification rules. For instance, a CompanyEmail might have to end in a company domain, while a personal email might be less restrictive.